test/rules/std_L048_test.py

```diff
@@ -0,0 +1,16 @@
+"""Tests the python routines within L048."""
+import sqlfluff
+
+
+def test__rules__std_L048_raised() -> None:
+    """L0048 is raised for quoted literals not surrounded by a single whitespace."""
```

Comment:
Nit (which is lucky as looking for a small change to kick off CI 😁):

```suggestion
    """L048 is raised for quoted literals not surrounded by a single whitespace."""
```